### DISCUSSION TITLE
Create a training set for lighting style

### DISCUSSION THREAD

**Discussion by Triceratops9** [TheLastBen/fast-stable-diffusion#1553] [General]
Hi there,
I'm a photographer. I use the same lighting set up every time for a corporate portrait client of mine and have over a hundred portrait images I can use as reference. I'd like to create a training data set so a subject can send me photos of themselves they shoot with their iphone and I can then create a portrait that looks like one I would make in person using a data set I create. 

Please help me with direction about instance images and and concept images and general strategy for making this work. 

Thank you!
T9

**Comment by TheLastBen** [comment]
Hi, for that you need only about 5-10 instance images and train the style like any other style, start with a small amount of steps, then increase gradually.

**Reply by Triceratops9** [reply]
Thank you!
I think what I'm also asking is how to train a style. (I'm pretty new) Is there a link you can share with a tutorial?
I really appreciate all the work you're doing.
L

**Reply by TheLastBen** [reply]
For a style you can simply run it with the default settings, it will take 10 minutes to train, test the model, then if not enough, resume the training of the UNet only with a learning rate of 2e-6.
Make sure you rename the instance images to one unique identifier which you will use in the prompt.
